activepieces/activepieces
# feat: added oauth2 props for frontend + saving them

## What does this PR do?
Address issue #590 



## Type of change


## How should this be tested?

Add an oauth2 property and and have its name within the auth url and the token url like this :
"https://{propertyName}.service-you-want-to-connect-to.com"

and once you click connect you should see it in the url.
